packages/compose-tunnel-agent/src/docker-proxy/index.test.ts

```diff
@@ -0,0 +1,248 @@
+import http from 'node:http'
+import net from 'node:net'
+import { describe, expect, beforeAll, afterAll, test, jest, it } from '@jest/globals'
+import { ChildProcess, spawn, exec } from 'child_process'
+import pino from 'pino'
+import pinoPretty from 'pino-pretty'
+import Dockerode from 'dockerode'
+import fetch from 'node-fetch'
+import { inspect, promisify } from 'node:util'
+import waitForExpect from 'wait-for-expect'
```

Comment:
wait-for-expect seems to be not maintained.
Might worth using alternative or just copying the implementation. (it's a single file)